### DISCUSSION TITLE
Add a Bev Minute event

### DISCUSSION THREAD

**Discussion by ebanner** [Virtual-Coffee/virtualcoffee.io#1160] [New Ideas]
Perhaps this is a really out there idea, but I was thinking about it

At my last company, we had this weekly event called Bev Minute. On Thursday at like 3:00 we all (50-100 of us) would get in a conference room. Ahead of time, there was a shareable google slides presentation where we could each add a slide with our name and something we thought was short and cool. Then we would go through the slide presentation and take turns getting up and doing short little presentations when our slide came up

It didn't have to be work related (but sometimes was) or tech related, just something you thought was cool and wanted to share with everyone. Maybe something you learned that week or even just some fun random thing

**Comment by BekahHW** [comment]
That sounds fun! On Apr 14, 2024, at 1:44 PM, Edward Banner ***@***.***> wrote:﻿
Perhaps this is a really out there idea, but I was thinking about it
At my last company, we had this weekly event called Bev Minute. On Thursday at like 3:00 we all (50-100 of us) would get in a conference room. Ahead of time, there was a shareable google slides presentation where we could each add a slide with our name if we and something we thought was short and cool. Then we would go through the slide presentation and take turns doing short little presentations
It didn't have to be work related (but sometimes was) or tech related, just something you thought was cool and wanted to share with everyone. Maybe something you learned that week or even some fun random thing
